### PR TITLE
Added single navigation bar functionality. 

### PR DESCRIPTION
**Pull request by satyamraghuwanshi136**
This PR solves issue #13 

### Things that I have done to achieve this:-

- Created custom element **"navbar.js"** file in the **"script"** directory. 
- Including this **"navbar.js"** file in all **.html** files using the **script tag**. 
- Adding the **`"<my-navbar></my-navbar>"`**  tag in the place of previous navbar code to include navbar.

@ggiande Thank you very much for letting me work on this issue.

**_let me know if I can help solve another issue. I will be happy to work._**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -171,7 +124,7 @@ <h3 class="mb-0">Events & Meetings</h3>
 
       <!-- Blog -->
       <article class="row g-5">
-        <div class="col-9">
+        <div class="col-sm-12 col-md-9">
```
Comment (ggiande) [inline index.html:127]: Unsure why this line is marked as modified as new... it's already checked into dev with the bootstrap mods

Comment (satyamraghuwanshi136) [reply index.html:127]: Yes. I am also not able to understand. What can I do now?

**Comment by ggiande** [comment]
Hello, issue #22 now blocks #20. Feel free to tackle that issue if you'd like. For the time being, this issue will be blocked by #22. Thanks for your patience.

**Comment by satyamraghuwanshi136** [comment]
Sure, I will work on the #22 issue first.

**Comment by satyamraghuwanshi136** [comment]
Hi @ggiande, I have pulled the changes, resolved the conflicts, and updated this PR.
Now, you can merge this.

Can you please tell me why I am not on the contributor's list of this project as my **#23 PR** got merged?

**Comment by ggiande** [comment]
Working on reviewing rn. I'm not sure; checkout out this [stack overflow post](https://stackoverflow.com/questions/40644933/not-showing-as-a-contributor-on-github) on this.

**Review by ggiande — APPROVED** [review]
LGTM!

**Pull request merged**